fileutil/checksum_test.go

```
@@ -0,0 +1,113 @@
+	assert.False(transferred)
+
+	err = checksum.SaveState()
+
+	defer func() {
+		if err := checksum.DeleteState(); err != nil {
+			t.Error(err)
+		}
+	}()
+
+	assert.Nil(err)
+	transferred, err = checksum.IsFileTransferred()
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)